grakn-engine/src/test/java/ai/grakn/engine/controller/GraqlControllerTest.java

```diff
@@ -169,6 +169,14 @@ public void whenRunningQueryWithLimitEmbedded_HalResponseIsTheSameAsJava() {
     }
 
     @Test
+    public void whenMatchingRules_ResponseStatusIs200() {
+        String queryString = "match $x sub rule; get;";
```

Comment:
Can you please change:

```
String queryString = "match $x sub rule; get;";
```

to 

```
String queryString = "match $x sub " + Schema.MetaSchema.RULE.getLabel().getValue() + "; get;";
```

I know it's SUPER long but it makes handling refactors SOOO much easier.